### PR TITLE
Verify title vernacular works

### PR DESCRIPTION
**Pull request by skng5**
Partially addresses # COB-143

### CODE REVIEW

**Comment by bibliotechy** [comment]
@sensei100 @skng5 Is this ready to go now?

**Comment by skng5** [comment]
@bibliotechy @sensei100 - We're going to address the rest of the items in #COB-143 first.

File: app/models/traject_indexer.rb [hunk 1]
```diff
@@ -69,7 +69,7 @@ def get_xml options={}
   acc.replace [acc.join(' ')] # turn it into a single string
 end
 
-to_field 'title_vern_display', extract_marc('245a', :trim_punctuation => true, :alternate_script=>:only)
+to_field 'title_vern_display', extract_marc('245a', :trim_punctuation => true, :alternate_script=>false)
```
Comment (bibliotechy) [inline app/models/traject_indexer.rb]: Try this with only , instead of false?

File: spec/features/record_page_fields_spec.rb [hunk 2]
```diff
@@ -0,0 +1,242 @@
+require 'rails_helper'
+require 'traject'
+require 'traject/command_line'
```
Comment (bibliotechy) [inline spec/features/record_page_fields_spec.rb]: Don't need traject requires here

**Comment by sensei100** [comment]
@bibliotechy done.

**Pull request merged**